app/Services/VoucherEvaluator/Evaluations/DeductFromCarer.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Services\VoucherEvaluator\Evaluations;
+
+use Carbon\Carbon;
+
+class DeductFromCarer extends BaseChildEvaluation
+{
+    public $reason = '';
+    private $specification;
+
+    /**
+     * DeductFromCarer constructor.
+     * @param int|null $value
+     */
+    public function __construct(Carbon $offsetDate = null, int $value = null)
+    {
+        parent::__construct($offsetDate, $value);
+    }
+
+    public function test($candidate)
+    {
+        parent::test($candidate);
+        $children = $candidate->family->children;
+        // This rule should only apply to the first 'child', who will be the carer.
+        return ($candidate->id === $children[0]->id)
```

Comment:
this checks if the carer ID is the same as the same as the first child's ID, which is almost certainly not the case. 
What this rule should do is check to see if the candidate (Family?) has at least one child.

we can
- rename the class to "FamilyHasOneOrMoreChildren"; this rule tests is a family has at least one child
- set the test to return `$candidate->has('children');`

When we program the DB with this rule, we apply a negative credit to it, and the system takes care of applying that when this rule is true.